src/Products/urban/exportimport.py

```diff
@@ -233,12 +234,20 @@ def addUrbanEventTypes(context):
     attribute = 'urbanEventTypes'
     module = __import__(module_name, fromlist=[attribute])
     urbanEventTypes = getattr(module, attribute)
+    attribute = 'REFNIS_2019'
+    module = __import__(module_name, fromlist=[attribute])
+    refNIS_2019 = getattr(module, attribute)
 
     site = context.getSite()
 
     log = []
     gslogger = context.getLogger('addUrbanEventTypes')
     tool = getToolByName(site, 'portal_urban')
+    matched_outsideDirection = None
+    for refNIS in refNIS_2019:
```

Comment:
Tu plutôt  écrire ça  du genre: 

directions_by_NIS = dict([(str(ref['Code INS']), ref['Directions extérieures') for ref in refNIS_2019])
matched_outsideDirection = directions_by_NIS.get(NIS, None)